docs/references/react/use-organizations.mdx

```diff
@@ -1,3 +1,169 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganizations
+title: useOrganizations()
+description: The useOrganizations() hook has been deprecated in favor of useOrganization().
 ---
+
+# `useOrganizations()`
+
+<Callout type="danger">
+  This hook has been deprecated in favor of [`useOrganization()`](/docs/references/react/use-organization) and [`useOrganizationList()`](/docs/references/react/use-organization-list).
```

Comment:
Our [Organizations Guide](/docs/organizations/overview) provides a detailed overview for working with Organizations.